File: c#-2/mostrarCompra.cs
using System;

class mostrarCompra 
{
  public static void Main (string[] args) 
  {
    string nome;
    double preco, total, quantidade;

    Console.WriteLine("Digite o nome do seu produto");
    nome = Convert.ToString(Console.ReadLine());

    Console.WriteLine("Digite o preço do seu produto");
    preco = Convert.ToDouble(Console.ReadLine());

    Console.WriteLine("Digite a quantidade de produtos");
    quantidade = Convert.ToDouble(Console.ReadLine());

    if(quantidade == 1) {
      Console.WriteLine("O " + nome + " o valor do produto foi " + preco);
    }
    else {
      total = preco * quantidade;
      Console.WriteLine("O " + nome + " o valor total da compra foi de " + total);
    }
  }
}

File: python-lista2/mostrarCompra.py
nome = input("Digite o nome do produto: ")
preco = input("Digite o preço do produto: ")
quantidade = input(f"Digite a quantidade de {nome} que você deseja: ")

if quantidade == 1:
  print(f"O valor total da compra de {nome} foi de {preco}")
else:
  total = float(preco) * float(quantidade)
  print(f"O valor total da compra de {quantidade} {nome} foi de {total}")